jerry-main/main-utils.c

```diff
@@ -201,6 +201,41 @@ create_test262 (jerry_value_t global_obj) /**< global object */
   return test262_object;
 } /* create_test262 */
 
+static void
+promise_callback (jerry_promise_event_type_t event_type, /**< event type */
+                  const jerry_value_t object, /**< target object */
+                  const jerry_value_t value, /**< optional argument */
+                  void *user_p) /**< user pointer passed to the callback */
+{
+  (void) value; /* unused */
+  (void) user_p; /* unused */
+
+  if (event_type != JERRY_PROMISE_EVENT_REJECT_WITHOUT_HANDLER)
+  {
+    return;
+  }
+
+  jerry_value_t reason = jerry_get_promise_result (object);
```

Comment:
Please keep the original port implementation: https://github.com/jerryscript-project/jerryscript/pull/4613/files#diff-25a64be3160f6b2ab557a9c399b8f8a0476d69057f7e7bfaa6fa5786624c3ba4L23 since it has a string length limit for the reason of the rejection.